akvo/rsr/tests/usecases/test_change_project_parent.py

```diff
@@ -37,8 +42,7 @@ def test_change_parent_to_sibling(self):
         # When
         command.change_parent(new_project.object, child_project.object)
         # Then
-        self.assertIsNone(new_project.object.parents_all().filter(id=root.object.id).first())
-        self.assertIsNotNone(new_project.object.parents_all().filter(id=child_project.object.id).first())
+        self.assertEqual(new_project.object.parent(), child_project.project)
```

Comment:
Failed when change to `self.assertEqual(Project.objects.get(title='New project').parent(), child_project.project)`

Reply:
Fixed in 086c9ca39ed0327d2c6b33a91903fd69b88318bf